### DISCUSSION TITLE
Optional `poolId` in `StakeDelegation` certificate interface

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#339] [Archives]
Hello.

What is the reason behind making `poolId` optional in the [StakeDelegation interface](https://github.com/CardanoSolutions/ogmios/blob/master/clients/TypeScript/packages/schema/src/index.ts#L584).

The [conway cddl](https://github.com/input-output-hk/cardano-ledger/blob/efda9c61ef2b75723ff52ee13ee5a16f20cbdb5f/eras/conway/test-suite/cddl-files/conway.cddl#L291C25-L291C25) shows it as mandatory.

Thank you!

**Comment by KtorZ** [comment]
It used to be missing from the Conway first iterations (on version prior to 8.3.0 if I recall correctly). 

This will be amended with the -rc3 release of Ogmios coming shortly.
